### ISSUE TITLE
[Panel]: Focus outline in responsive mobile

### ISSUE THREAD

**Issue by Martin-Rippel** [UI5/webcomponents#12811]
### Bug Description

When pressing on the header area of a collapsible panel without a toolbar in responsive mobile, a focus outline is displayed for the header area. However, it’s common for interactive elements in responsive mobile to not receive a focus outline when a user interacts with them. Therefore, this focus outline should be removed for panel headers in responsive mobile.
This issue can be reproduced in the [UI5 Web Components samples](https://ui5.github.io/webcomponents/components/Panel/) using a mobile device.

### Steps to Reproduce

1.
2.
3.
...

### Priority

None

### UI5 Web Components Version

2.17.0

### Browser

Safari

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by IlianaB** [comment]
Hello @UI5/ui5-team-ctr-ril , I am forwarding this issue to you, because visual focus outline on Panel component is shown on mobile devices (tested with iPhone), while it should not be.

Regards,
Iliana

**Comment by aleksandar-terziev** [comment]
Hello Martin,

We have created a BLI - [BGSOFUIRILA-4208](https://jira.tools.sap/browse/BGSOFUIRILA-4208) on our board, therefore this issue can be closed
